content/source/docs/cloud/workspaces/managing-variables.html.md

```diff
@@ -57,6 +72,17 @@ To delete a variable:
 1. Click the ellipses next to the variable you want to delete and select **Delete**.
 1. Click **Yes, delete variable** to confirm your action.
 
+## Loading Variables from Files
+
+You can set [Terraform variable](/docs/cloud/workspaces/variables.html#terraform-variables) values by providing any number of [files ending in `.auto.tfvars`](/docs/language/values/variables.html#variable-files) to workspaces that use Terraform 0.10.0 or later. When you trigger a run, Terraform automatically loads and uses the variables defined in these files. If any variable from the workspace has the same key as a variable in the file, the workspace variable overwrites variable from the file.
+
+You can only do this with files ending in `auto.tfvars`; Terraform Cloud does not automatically recognize other types of `.tfvars` files. You can only apply other types of `.tfvars` files [on the command line](#run-specific-variables) for each run.
+
+~> **Note:** Terraform Cloud loads variables from files for each Terraform run, but does not automatically persist those variables to the Terraform Cloud workspace or display them in the **Variables** section of the workspace UI.
+
+~> **Note:** Variable values applied using `terraform.tfvars` are ignored by Terraform Cloud
```

Comment:
It can be visually overwhelming to have two notes in a row. I added a suggestion above where we can specifically call out that `terraform.tfvars` won't work this way. Does that feel okay to you? If so, would you feel comfortable deleting this extra note?